package.json

```diff
@@ -29,7 +29,7 @@
     "@graphql-tools/merge": "^9.0.19",
     "@graphql-yoga/plugin-response-cache": "^3.13.0",
     "@hypercerts-org/contracts": "2.0.0-alpha.12",
-    "@hypercerts-org/marketplace-sdk": "0.3.37",
+    "@hypercerts-org/marketplace-sdk": "0.4.2",
```

Comment:
aren't we at 0.5.0?

Reply:
We're at 0.5.0-alpha still, but we could release that first and then rebase this PR.

Reply:
Addressed